doctr/models/zoo.py

```diff
@@ -39,11 +49,18 @@ def ocr_predictor(
         >>> out = model([input_page])
 
     Args:
-        arch: name of the architecture to use ('db_sar_vgg', 'db_sar_resnet', 'db_crnn_vgg', 'db_crnn_resnet')
+        det_arch: name of the detection architecture to use ('db_resnet50', 'db_mobilenet_v3_large')
+        reco_arch: name of the recognition architecture to use ('crnn_vgg16_bn', 'sar_resnet31')
         pretrained: If True, returns a model pre-trained on our OCR dataset
+        assume_straight_pages: if you only pass straight pages with straight boxes, activate this arg.
+            to speed up the pipeline!
```

Comment:
"if True, speeds up the inference by assuming you only pass straight pages without rotated textual elements." :+1: